.github/workflows/updates/ReaPack.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
As we're not adding any app named ReaPack, does this update script still belong here?

Reply:
No, I guess I missed that too... lol